### ISSUE TITLE
Alias wrapper - multiple aliases with { }

### ISSUE THREAD

**Issue by BarbzYHOOL** [fish-shell/fish-shell#5038]
fish 2.7.1

In bash, we can do this:

    alias {my_alias,my_alias_2}="my_command"

So it makes 2 aliases for the same command (can be as many aliases as wanted).

It's not working in fish and is there an alternative?

**Comment by faho** [comment]
The alternative in fish as in bash is to just define two aliases:

```fish
alias my_alias="my_command"
alias my_alias_2="my_command"
```

**Comment by BarbzYHOOL** [comment]
Not practical

I also looked at abbr but it is suggested to use functions to store in the config/functions dir https://superuser.com/questions/1049368/add-abbreviations-in-fish-config 

What are the downsides of using abbr?

**Comment by faho** [comment]
>Not practical

If you say so...

>What are the downsides of using abbr?

abbrs are a purely interactive feature. They don't become part of the language in the way that functions do (and in fish, there are no aliases, only functions - see `type alias`). It just allows you to type something and get something else into the commandline.

Which I'd argue makes them _better_ than aliases.

**Comment by mqudsi** [comment]
I completely agree with @faho on this one, _but_ this actually brings up a more general concept I've missed in the past.

```fish
~> echo {foo, bar}
```

This expands to a single command with two arguments, `foo` and `bar`. 

I have oftentimes found myself wishing fish had a shorthand that would expand to _n_ **commands** instead of _n_ **arguments**, so that `echo {foo, bar}` would be equivalent to `echo foo; echo bar;`, or more universally, equivalent to `for x in EXPR; cmd $x; end;`

Basically, an expansion-based shorthand for a `for` loop.
